guides/installation/devenv.md

```diff
@@ -114,13 +114,6 @@ Before booting up your development environment, configure Cachix to use Shopware
 cachix use shopware
 ```
 
-By default, `shopware/platform` uses unfree software like Blackfire. To be able to use unfree software, you have to *allow* it:
```

Comment:
@shyim This shouldn't be necessary anymore since it's configured in `devenv.yaml`, is it?